modules/virt-exposing-and-creating-mediated-devices.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: PROCEDURE
+[id="virt-exposing-and-creating-mediated-devices_{context}"]
+= Exposing and creating mediated devices
+
+You can expose and create mediated devices such as virtual GPUs (vGPUs) by editing the `HyperConverged` custom resource (CR).
+
+.Prerequisites
+
+* You enabled the IOMMU (Input-Output Memory Management Unit) driver.
+
+.Procedure
+
+. Edit the `HyperConverged` CR in your default editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Add the mediated device information to the `HyperConverged` CR `spec`, ensuring that you include the `mediatedDevicesConfiguration` and `permittedHostDevices` stanzas. For example:
++
+.Example configuration file
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+  namespace: openshift-cnv
+spec:
+  mediatedDevicesConfiguration:
+    mediatedDevicesTypes: <1>
+    - nvidia-231
+    nodeMediatedDeviceTypes: <2>
+    - mediatedDevicesTypes: <3>
+      - nvidia-233
+      nodeSelector:
+        kubernetes.io/hostname: node-11.redhat.com
+  permittedHostDevices: <4>
+    mediatedDevices:
+    - mdevNameSelector: GRID T4-2Q
+      resourceName: nvidia.com/GRID_T4-2Q
+    - mdevNameSelector: GRID T4-8Q
+      resourceName: nvidia.com/GRID_T4-8Q
+...
+----
+<1> This configuration is applied globally.
+<2> Optional: override the global configuration for specific nodes.
+<3> This field is required if you use `nodeMediatedDeviceTypes`.
+<4> A list of host devices that are permitted to be used in the cluster, including Peripheral Component Interconnect (PCI) and mediated devices.
```

Comment:
Feel, this should be updated to, `A list of mediated Devices that are permitted to be used in the cluster."

As, IIUC, For vGPU, we should not refer to it as "host devices".